### DISCUSSION TITLE
Word Boosting Support | ASR

### DISCUSSION THREAD

**Discussion by naderessam110** [espnet/espnet#5191] [Q&A]
Hi,

is there any support for word boosting in ESPnet2 ASR right now or in the future for increasing the chance of recognition for a provided list of keywords?

Thanks

**Comment by sw005320** [comment]
I don't know the actual techniques for word boosting.
But anyway, we do not support such a function in the official repo, but we're developing several methods internally.
If there are many requests, we will consider the priority of this item.
